COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@jagajaga Thanks for the suggestion but I am doing this at home in my free time. Can you elaborate why the notice is unacceptable to you? Do you believe it would be factually incorrect?